realm/realm-jni/src/io_realm_internal_tableview.cpp

```diff
@@ -131,6 +131,18 @@ JNIEXPORT void JNICALL Java_io_realm_internal_TableView_nativeClose(
     delete TV(nativeViewPtr);
 }
 
+JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeGetCopy(
+    JNIEnv* env, jobject, jlong nativeViewPtr)
+{
+    if (!VIEW_VALID_AND_IN_SYNC(env, nativeViewPtr))
+        return 0;
+    try {
+        TableView* copiedTableView = new TableView(*TV(nativeViewPtr));
```

Comment:
I doubt that doing a new `find_all` would be accurate. The TableView might be restricted by some query. I am pretty sure that copying a TableView will also copy any parent views/queries, but @rrrlasse / @finnschiermer can probably confirm that.